CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 #CHANGE LOG
 
+##Version 0.?
+
++ DSL Improvements
+  + Improved Documentation Style - The infile documentation of flows and operations has been changed to be more structured.
+	+ System Properties Files - System properties files now end with the .sl extension and their syntax has changed.
++ CLI Enhancements
+  + Inspect - The new `inspect` command displays a file's documentation including description, inputs, outputs and results.
+	+ Default Folder - The location of the default folders for automatically importing system properties files and input files has moved to `cslang/configuration/inputs` and `cslang/configuration/properties`.
```

Comment:
we can also mention config/cslang.properties with log4j and encoding props (by default log4j can be customized from configuration.logging/log4j.properties)